v2/cn/events.md

```diff
@@ -0,0 +1,15 @@
+<!---
+    @title         Events
+    @creator       Yichun Zhang
+    @modifier      Yichun Zhang
+    @created       2016-11-28 15:58 GMT
+    @changes       1
+--->
+
+
+#  国外活动
+
+#  国内活动
+* OpenResty Con 2016 (2016年12月10日 9:00AM @深圳)
```

Comment:
maybe it should be `9:00 AM` rather than `9:00AM`.
So http://con.openresty.org/cn/2016/ alos should be  `9:00 AM`